dist.ini

```diff
@@ -4,4 +4,4 @@ author = Hans Zandbelt (@zandbelt)
 is_original = yes
 license = apache2
 repo_link = https://github.com/zmartzone/lua-resty-openidc
-requires = openresty, pintsized/lua-resty-http >= 0.08, bungle/lua-resty-session >= 2.8, cdbattags/lua-resty-jwt >= 0.2.0, jkeys089/lua-resty-hmac >= 0.2
+requires = openresty, pintsized/lua-resty-http >= 0.08, bungle/lua-resty-session >= 2.8, cdbattags/lua-resty-jwt >= 0.2.0
```

Comment:
ok, I did not mean to remove it from the OPM dist.ini: AFAIK that format doesn't have the option to explicitly include dependencies